checker/tests/nullness/JavaCopExplosion.java

```diff
@@ -3,6 +3,7 @@
 
 @org.checkerframework.framework.qual.DefaultQualifier(Nullable.class)
 class Explosion {
+    // :: error: (declaration.inconsistent.with.extends.clause)
```

Comment:
This error seems to be about 
```
class @Nullable ExplosiveException extends @Nullable Exception {}
```
What is wrong with that declaration?

Reply:
The error message is: ```Class @Nullable cannot extend @NonNull```.
This suggests that the default is not getting applied to extends clauses.